docs/developer-docs/integrations/sns/launching/launch-summary.md

```diff
@@ -0,0 +1,332 @@
+
+  This includes adding the SNS root canister as the controller of the dapp and removing
+  yourself (and possibly other developers) from the list of
+  controllers. Also, this includes "registering" the dapp with the SNS so that SNS
+  root is aware that it controls these canisters.
+
+  The registration is done by **SNS proposal**.
+  
+  As there are already initial neurons (from Step 3), this,
+  as well as potential upgrades to the dapp, can be realized by SNS proposals and by the
+  majority of initial neurons voting in favor of them.
+  The initial neurons cannot do other things, such as changing
+  the SNS  parameters, as the SNS governance canister is still in
+  pre-decentralization-swap mode.
+
+  :::info
+  Note that this is an *SNS* proposal, not an NNS proposal, even though many users would vote on it within [the NNS frontend dapp](nns.ic0.app).
+  :::
+
+  If successful, at the end of stage, we have:
+
+
+|  Decentralization swap state                      | PENDING                                    |  
+|-------------------                                | ----------------                        |
+
+
+|  Objects in an app subnet                                      | State                    |  Controlled by |
+|-------------------                                | ----------------                      | ----------------|
+| A dapp                                            | Operational                           | Dapp developer principal | 
+
+|  Objects in the SNS subnet                                       | State                  |  Controlled by |
+|-------------------                                | ----------------                      | ----------------|
+| ~~A principal that can deploy to SNS subnet~~     | Revoked because it is 1-time use      | NA                        |
+| Newly-created SNS canisters on the SNS subnet     | Pre-decentralization swap mode        | Initial developer neurons |  
+| Initial developer neurons                         | Pre-decentralization swap mode        | Dapp developer principal  |
+| Treasury account on the SNS Ledger                | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+| Swap account on the SNS Ledger                    | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+
+
+- #### Step 7: The initial SNS developer neurons accepts or rejects the SNS proposal
+
+  The initial SNS developer neurons are declared in the initial parameters and available at SNS installation.
+
+  This is the **second of three** proposals that need to successfully pass for the process to continue.
+
+  Since this proposal is passed by the SNS DAO. When one owns SNS neurons or votes on SNS proposals, one is are part of the SNS DAO. It is common for people to vote on both NNS and SNS proposals from within the [NNS Frontend dapp](nns.ic0.app), but they are two separate DAOs.
```

Comment:
Agree with content, but think the first part might not be a correct phrase yet? 
(Not going into details as I assume this might be cleaned up in next round)